### PR TITLE
Issue 25071: favorite pages update cards and add star

### LINKED ISSUE #25071: Favorites Pages: Update Cards and Add Star to Bookmarks Panel
### User Story
As a dotCMS stakeholder, I want to be able to update the cards to add more spacing to the content and add a star to the Bookmarks panel title, so I can have a better visual experience when using the portlet.

### Acceptance Criteria
The cards should:
- Remove the stars for each
- Reduce the padding to 1rem

The panel should:
- Add the star to the title so it reads: ⭐ Bookmarks

### Proposed Objective
Core Features

### Proposed Priority
Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.:
N/A

### Assumptions & Initiation Needs
N/A

### Quality Assurance Notes & Workarounds
N/A

### Sub-Tasks & Estimates:
N/A

### PR DESCRIPTION
**Pull request by danielmdob**
### Proposed Changes
* Remove star symbol for editing bookmark in pages panel
* Add star symbol besides checklist

### Checklist
- [X] Tests
- [X] Translations
- [X] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
image |  image image

### CODE REVIEW

**Comment by danielmdob** [comment]
Converted to draft while design decisions are being made.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss [hunk 1]
```diff
@@ -86,6 +87,35 @@
...
+    .pi-star-fill {
+        color: $color-palette-primary;
+        margin-right: $spacing-1;
+    }
+
+    .dot-pages-panel__header {
+        display: flex;
+        align-items: center;
+        margin-left: $spacing-3;
+        font-weight: 400;
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss:100]: We have a global variable for this, you can check this folder `core-web/libs/dotcms-scss` there we have all the styles we use and this one you can find it on `core-web/libs/dotcms-scss/shared/_fonts.scss` it is exactly `$font-weight-regular-bold`

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.spec.ts [hunk 2]
```diff
@@ -255,13 +255,17 @@ describe('DotPagesFavoritePanelComponent', () => {
...
+            expect(elem.componentInstance['iconPos']).toBe('end');
             expect(elem.componentInstance['expandIcon']).toBe('pi pi-angle-down');
             expect(elem.componentInstance['collapseIcon']).toBe('pi pi-angle-up');
-            expect(elem.componentInstance['header']).toBe('favoritePage.panel.header');
             expect(elem.componentInstance['toggleable']).toBe(true);
         });
 
+        it('should have a star icon in the header', () => {
+            const elem = de.query(By.css('.dot-pages-panel__header'));
+            expect(elem.nativeElement.querySelector('.pi-star-fill')).toBeTruthy();
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.spec.ts:266]: add a `data-testId` to the icon and query it using that, using style classes can lead to more work in the future if we want to change that class or that icon

**Review by zJaaal — COMMENTED** [review]
.

File: core-web/.nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-v16.13.2
\ No newline at end of file
+v18.14.2
```
[the remark this answers is not in the document]
Comment (fmontes) [reply core-web/.nvmrc:1]: Let's remove until we figure it out what's going on in your env.

**Review by fmontes — CHANGES_REQUESTED** [review]

**Comment by danielmdob** [comment]
@zJaaal @fmontes thanks for the feedback, just re-requested review.

**Review by zJaaal — APPROVED** [review]

**Comment by danielmdob** [comment]
I chose the wrong base, closing.

**Pull request closed without merging**